src/test/java/id/fahreza/marsrover/MarsRoverTest.java

```diff
@@ -6,128 +6,95 @@
 public class MarsRoverTest {
 
     @Test
-    public void testInitMaxCoordinate_givenUpperRightCoordinate_shouldReturnUpperRightCoordinate() {
-        int x = 3;
-        int y = 3;
-        MarsRover marsRover = new MarsRover(x, y);
-        Assert.assertEquals(x, marsRover.getMaximumX());
-        Assert.assertEquals(y, marsRover.getMaximumY());
+    public void constructor_givenUpperRightCoordinate_shouldSetMamaximumXimumXY() {
```

Comment:
Please fix the typo

Reply:
Done, already fixed on the next commit. Please review it again.